packages/icon/src/Icon.ts

```diff
@@ -70,14 +70,28 @@ export class Icon extends IconBase {
         }
     };
 
+    private announceIconImageSrcError(): void {
+        this.dispatchEvent(
+            new Event('error', {
+                cancelable: true,
+                bubbles: true,
+                composed: true,
```

Comment:
The default values for the event options in the event that you are redispatching here are all `false`. Can you share a little what you expect to do with these value as `true`?

Specifically, you've not added any code to support an application interacting with the `cancelability` of the event, but I'm not sure even what you could do there is you decided to. Taking it out might be the right call.

For `bubbles` and `composed`, is being able to hear this message outside of the shadow root where the `<sp-icon>` element is that useful to a consuming developer? Answers here can help form the basis for a little documentation around this addition so that this feature can be well leveraged by others.

Reply:
I made the silly mistake of copy-pasting this code and setting these back to default again, let me know if you have any more suggestions, Thanks